test/system/invitation_lists_test.rb

```diff
@@ -0,0 +1,64 @@
+require "application_system_test_case"
+
+class InvitationListsTest < ApplicationSystemTestCase
+  if bulk_invitations_enabled?
```

Comment:
This test isn't running because we default to this feature being turned off. I think this test should change the config setting prior to test setup to ensure that this test _always_ run. (And it should restore the original setting afterwards.)